@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -491,11 +491,11 @@ export class PoiManager {
             const fadeNear =
                 technique.fadeNear !== undefined
                     ? getPropertyValue(technique.fadeNear, displayZoomLevel)
-                    : 0;
+                    : technique.fadeNear;
```

Comment:
I think it should be rather 0 or even better defaultRenderParams.fadeNear, otherwise you will assign undefined here (intentionally?).